### DISCUSSION TITLE
Any change of a new release for dockertest (e.g. v3.9.2)?

### DISCUSSION THREAD

**Discussion by LinuxChristian** [ory/.github#60] [Q&A] [answered]
Hi,

I can see that the previous release of https://github.com/ory/dockertest was May last year (v 3.9.1). That release ships _opencontainers/runc v1.1.2_ which has a [know security issue](https://nvd.nist.gov/vuln/detail/CVE-2023-28642) that is fixed in _v1.1.5_ which is the current version in master.

I know I could just point _go mod_ towards a specific commit on master but it's just cleaner with an official release 🙂 Any plans to ship dockertest v3.9.2 soon?

Thanks,
Christian

**Comment by LinuxChristian** [comment]
@aeneasr, I can see you created a new release the last time there was a security issue with runc (https://github.com/ory/dockertest/issues/357) in dockertest. Any change of getting a new release with the security fix from runc v1.1.5.

**Reply by aeneasr** [reply, marked as answer]
thanks - done

**Reply by LinuxChristian** [reply]
Fantastic - thanks a lot
